src/openssl/ssl/server.cr

```diff
@@ -35,6 +35,9 @@ class OpenSSL::SSL::Server
   # Returns `true` if this SSL server has been closed.
   getter? closed : Bool = false
 
+  # When `true` a call to `#accept` will also initiate the SSL handshake.
+  property start_immediately : Bool = true
```

Comment:
```suggestion
  property? start_immediately : Bool = true
```

Reply:
I don't think every boolean property must have a question mark. In particular I like to use it when is a read only property or with methods. Also, I copied this name from Ruby's OpenSSL and I prefer to keep the same name.